src/Message.ts

```diff
@@ -164,14 +164,14 @@ export default class Message implements proto.MessageV1 {
     if (!header.recipient.preKey) {
       throw new Error('missing message recipient pre-key')
     }
-    const recipient = new PublicKeyBundle(
-      new PublicKey(header.recipient.identityKey),
-      new PublicKey(header.recipient.preKey)
-    )
-    const sender = new PublicKeyBundle(
-      new PublicKey(header.sender.identityKey),
-      new PublicKey(header.sender.preKey)
-    )
+    const recipient = new PublicKeyBundle({
```

Comment:
I've changed the constructor to take the proto, to save duplicate undefined checking. That's what we do everywhere else too.